core/modules/filters.js

```diff
@@ -331,7 +331,7 @@ exports.compileFilter = function(filterString) {
 		})());
 	});
 	// Return a function that applies the operations to a source iterator of tiddler titles
-	var fnMeasured = $tw.perf.measure("filter: " + filterString,function filterFunction(source,widget) {
+	var fnMeasured = $tw.perf.measureFn("filter"+filterString,filterString,function filterFunction(source,widget) {
```

Comment:
make the name clear

Reply:
I think the text-prefix `filter` should be removed completely. It only makes automatic post-processing of the data more complicated.

Reply:
Why has the prefix been changed to remove the colon? There is a backwards compatibility impact so we shouldn't change the text unless there's a good reason.

Removing the space around the "+" breaks our coding standards.